docs/ref/models/database-functions.txt

```diff
@@ -149,6 +149,25 @@ will result in a database error.
     The PostgreSQL behavior can be emulated using ``Coalesce`` if you know
     a sensible maximum value to provide as a default.
 
+``NullIf``
+----------
+
+.. class:: NullIf(expression1, expression2)
+
+.. versionadded:: 2.1
+
+Accepts two arguments, and return ``NULL`` if they are equal, otherwise
```

Comment:
return -> returns
NULL or None? (add an example for that case?)
otherwise returns expression1